docs/content/topics/operators/gateways.md

```diff
@@ -11,340 +11,518 @@ aliases:
-
-  # The type of event. This field is freeform. Brigade does not have a list of
-  # pre-approved event names. Thus, you can define your own event_type
-  event_type: push
-
-  # Revision describes a vcs revision.
-  revision:
-
-    # Commit is the commitish/reference for any associated VCS repository. By
-    # default, this should be `master` for Git.
-    commit: 6913b2703df943fed7a135b671f3efdafd92dbf3
-
-    # Ref is the symbolic ref name. (refs/heads/master, refs/pull/12/head, refs/tags/v0.1.0)
-    ref: master
-
-  # This should be the same as the `name` field on the secret
-  build_name: example
-
-  # This should be the same as the 'project' label
-  project_id: brigade-1234567890
-
-  # This should be the same as the 'build' label
-  build_id: 01C1R2SYTYAR2WQ2DKNTW8SH08
-
-  # The payload can contain arbitrary data that will be passed to the worker
-  # JavaScript. It is passed to the script unparsed, and the script can parse
-  # it as desired.
-  payload: "{ 'foo': 'bar' }"
-
-  # An event can supply a script to execute. If it does not supply a script,
-  # Brigade will try to locate a 'brigade.js' file in the project's source
-  # code repository using the commit provided above.
-  script: "console.log('hello');"
+  created: "2021-08-11T22:22:41.366Z"
+  id: 48c960eb-5823-46d0-8390-ec6a2a966b98
+projectID: hello-world
+source: brigade.sh/cli
+type: exec
+worker:
+  spec:
+    configFilesDirectory: .brigade
+    defaultConfigFiles:
+      brigade.js: |
+        console.log("Hello, World!");
+    logLevel: DEBUG
+    useWorkspace: false
+    workspaceSize: 10Gi
+  status:
+    apiVersion: brigade.sh/v2-beta
+    ended: "2021-08-11T22:22:49Z"
+    kind: WorkerStatus
+    phase: SUCCEEDED
+    started: "2021-08-11T22:22:41Z"
 ```
 
-Again, note that any fields in the `data:` section above are shown cleartext,
-though in reality you _must_ base-64 encode them.
-
-The easiest way to create a secret like the above is to do so with the `kubectl`
-command, though there are a host of language-specific libraries now for creating
-secrets in code.
+Let's look at the high-level sections in the event definition above.  They are:
+
+  1. Event metadata, including:
+    i. The `apiVersion` of the schema for this event
+    ii. The schema `kind`, which will always be `Event`
+    iii. The `id` of the Event
+    iv. A `created` timestamp for the Event
+    v. The `projectID` that the Event is associated with
+    vi. The event `source`
+    vii. The event `type`
+  2. The `worker.spec` section, which contains worker configuration inherited
+    from the project definition associated with the event in combination with
+    system-level defaults.
+  3. The `worker.status` section, which contains the `started` and `ended`
+    timestamps and current `phase` of the worker handling the event. In the
+    example above, it has already reached the terminal phase of `SUCCEEDED`.
+
+To explore the current SDK definitions of an Event object, see the
```

Comment:
Nit: I'd drop "current" because it subtly suggests that things are subject to change, but we're avoiding breaking changes at all costs now.